Web3/Classes/PromiseKit/Web3+PromiseKit.swift

```diff
@@ -281,11 +275,6 @@ public extension Web3.Eth {
 fileprivate extension Web3Response {
 
     fileprivate func sealPromise(seal: Resolver<Result>) {
```

Comment:
I think we can now make `sealPromise` public as this implementation is really ok.